server/src/test/python/cromwell-az-e2e-test/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "cromwell-az-e2e-test"
+version = "0.1.0"
+description = ""
+authors = ["JVThomas <jthomas0209@gmail.com>"]
```

Comment:
Totally up to you, but do you want your @broadinstitute.org email address here?

Reply:
Ah, it's using my Github associated email. If there's a batch-team GH identity/email I'd rather put that instead.